common/thorhelper/thorhelper.hpp

```diff
@@ -28,6 +28,10 @@
  #define THORHELPER_API DECL_IMPORT
 #endif
 
+#define SLAVEIDSTR "SLAVEID"
```

Comment:
perhaps for greater clarify that this is a subst placeholder it could be "#SLAVEID#"

Reply:
Change to #define SLAVEIDSTR "#SLAVEID#" ?

Reply:
yes, slightly clearer that something special (to replace) I think.